e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -331,38 +331,12 @@ test('Example Imagery in Display layout', async ({ page }) => {
 
         return newImageCount;
     }, {
-        message: "verify that new images still stream in",
+        message: "verify that old images are discarded",
         timeout: 6 * 1000
-    }).toBeGreaterThan(imageCount);
+    }).toBe(imageCount);
 
     // Verify selected image is still displayed
     await expect(selectedImage).toBeVisible();
-
-    // Unpause imagery
```

Comment:
this test requires some open loop timing to work. specifically it can take up to 20s for the image to refresh here...

Reply:
@scottbell that's ok for the e2e tests as long as we've tagged them as slow

Reply:
Gotcha

Reply:
filed as https://github.com/nasa/openmct/issues/5374